spec/models/project_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe Project, type: :model do
+	describe 'model tests' do
+		before(:each) do
+			@user = create(:user, is_missing: true)
```

Comment:
we don't really use instance variables in tests. Use `let` instead

Reply:
Fixed